src/plugins/workspace/public/components/workspace_form/use_workspace_form.ts

```diff
@@ -80,6 +80,7 @@ export const useWorkspaceForm = ({
   const numberOfChanges = defaultValuesRef.current
     ? getNumberOfChanges(formData, {
         ...defaultValuesRef.current,
+        name: defaultValuesRef.current.name || '',
```

Comment:
Shall we update the type definition of `getNumberOfChanges` to support partial form data? Then we don't need to fallback to an empty name.

Reply:
Thanks for your suggestions!